Source/FakeItEasy.Specs/ConstructorWithVirtualCallsSpecs.cs

```diff
@@ -65,38 +92,165 @@ public class when_faking_a_class_which_calls_virtual_member_in_constructor_withi
+    }
 
-        It should_return_the_configured_value_from_the_scope_inside_and_outside_of_the_constructor_call = () =>
+    public class when_configuring_a_method_called_by_a_constructor_to_be_strict
+    {
+        static Exception exception;
+
+        Because of = () => exception = Record.Exception(() => A.Fake<MakesVirtualCallInConstructor>(options => options.Strict()));
+
+        private It should_throw_an_exception_during_the_constructor = () =>
         {
-            fake.VirtualMethodValueDuringConstructorCall.Should().Be("configured value in fake scope");
-            virtualMethodValueInsideOfScope.Should().Be("configured value in fake scope");
+            var expectedMessage = new[]
+            {
+                string.Empty,
+                "  Failed to create fake of type \"FakeItEasy.Specs.MakesVirtualCallInConstructor\".",
+                string.Empty,
+                "  Below is a list of reasons for failure per attempted constructor:",
+                "    No constructor arguments failed:",
+                "      No usable default constructor was found on the type FakeItEasy.Specs.MakesVirtualCallInConstructor.",
+                "      An exception of type FakeItEasy.ExpectationException was caught during this call. Its message was:",
+                "      Call to non configured method \"VirtualMethod\" of strict fake.",
+                "*"
+            }.AsTextBlock();
+
+            exception
+                .Should()
+                .BeAnExceptionOfType<FakeCreationException>()
+                .WithMessage(
+                    expectedMessage);
         };
+    }
 
-        It should_return_default_value_outside_of_scope = () => 
-            virtualMethodValueOutsideOfScope.Should().Be(string.Empty);
+    // This spec proves that we can configure methods of a strict fake and it applies to calls in the constructor (the 
+    // counterpart to the previous scenario with an un-configured strict fake):
+    public class when_configuring_a_method_called_by_a_constructor_to_be_strict_and_adding_configuration
+    {
+        static MakesVirtualCallInConstructor fake;
+
+        Because of = () =>
+            fake = A.Fake<MakesVirtualCallInConstructor>(options => options
+                .Strict()
+                .ConfigureFake(f => A.CallTo(() => f.VirtualMethod(A<string>._))
+                    .Returns("configured value of strict fake")));
+
+        It should_return_the_configured_value_during_the_constructor =
+            () => fake.VirtualMethodValueDuringConstructorCall.Should().Be("configured value of strict fake");
+
+        It should_return_the_configured_value_after_the_constructor =
+            () => fake.VirtualMethod(null).Should().Be("configured value of strict fake");
     }
 
-    public class when_faking_a_class_which_calls_virtual_property_members_in_constructor
+    public class when_wrapping_an_object_in_a_fake_that_calls_virtual_member_in_constructor

```

Comment:
The idea of the "ConstructorWithVirtualCallsSpecs" was to be a specification for *the new feature "support for mocked virtual calls during the c'tor"* (which could also get a new wiki page). All the scenarios for the different configuration options are just there to prove that all them *also work during the c'tor*.

I'm with you that that there should be separate "FakeScope/FakeConfigurator/FakeOptions" specs (which would duplicate some of the scenario(names) of `ConstructorWithVirtualCallsSpecs`). But personally I think that the scenarios in `ConstructorWithVirtualCallsSpecs` have a high cohesion and should not be separated (except the mentioned "feature intersection" scenarios).